src/main/java/lebron/Parser.java

```diff
@@ -0,0 +1,54 @@
+package lebron;
+
+import java.time.LocalDate;
+
+public class Parser {
+
+    public Command parse(String userInput) throws LeBronException {
+        String[] words = userInput.split(" ", 2);
+        String commandWord = words[0];
+
+        switch (commandWord) {
+        case "bye":
+            return new ExitCommand();
+        case "list":
+            return new ListCommand();
+        case "delete":
+            int index = Integer.parseInt(words[1]);
+            return new DeleteCommand(index);
+        case "mark":
+            index = Integer.parseInt(words[1]);
+            return new MarkCommand(index);
+        case "unmark":
+            index = Integer.parseInt(words[1]);
+            return new UnmarkCommand(index);
+        case "find":
+            String keyword = words[1];
+            return new FindCommand(keyword);
+        case "todo":
+            ToDos todo = new ToDos(words[1].trim());
+            return new AddCommand(todo);
+        case "deadline":
+            String[] splStr = words[1].split("/by ", 2);
```

Comment:
Perhaps `splStr` should be changed to a more descriptive variable name?

Reply:
I second this 👍